src/sentry/incidents/subscription_processor.py

```diff
@@ -384,6 +386,17 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             metrics.incr("incidents.alert_rules.skipping_already_processed_update")
             return
 
+        if features.has(
+            "organizations:workflow-engine-metric-alert-processing",
+            self.subscription.project.organization,
+        ):
+            # NOTE: feed the data through the new pipeline, but don't do anything with it yet.
+            # This will eventually change.
+            data_packet = DataPacket[QuerySubscriptionUpdate](
+                query_id=self.subscription.id, packet=subscription_update
+            )
+            process_data_packets([data_packet], "snuba_query_subscription")
```

Comment:
nit; i'd recommend registering the `snuba_query_subscription` as the `DATA_SOURCE_TYPE` in `incidents/types.py` or `incidents/grouptypes.py` (whichever makes more sense to you) that way you can share this text field around easily.

Reply:
commenting to keep an eye on this cause I'll use it in my PR too